### PR TITLE
[DOC-3961] [PL-46559] Update build custom delegate images

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update build custom delegate images

For reviewers, a preview is available:

https://65b05fdddb2ec617b7b950e2--harness-developer.netlify.app/docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile#build-a-custom-image-with-non-root-access-that-includes-custom-certificates

## What Type of PR is This?

- [ ] Issue
- [x] Feature
- [ ] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile.md [hunk 1]
```diff
@@ -81,19 +81,123 @@ To build your custom delegate image, do the following:
...
+
+   USER 1001
    
    ```
    
    This copies all the certificates from the local `./my-custom-ca` directory to `/opt/harness-delegate/my-ca-bundle/` directory inside the container.
 
-   :::info caution
+   :::info warning
    Don't copy your certificates to the folder `/opt/harness-delegate/ca-bundle` folder. This folder is reserved for storing additional certificates to install the delegate.
```
Comment (xingchi-jin) [inline docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile.md:100]: Can we put a link to https://developer.harness.io/docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs/#install-with-custom-certificates , so customers will know why this folder is reserved

**Review by xingchi-jin — APPROVED** [review]
approve with a nit

File: docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile.md (hunk 1 above)
Comment (brian-f-harness) [reply docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile.md:100]: Will add. Thanks @xingchi-jin.

Comment (brian-f-harness) [inline docs/platform/delegates/manage-delegates/build-custom-images-delegate-dockerfile.md:100]:
```suggestion
   Don't copy your certificates to the folder `/opt/harness-delegate/ca-bundle` folder. This folder is reserved for storing additional certificates to install the delegate. For more information, go to [Install with custom certificates](/docs/platform/delegates/secure-delegates/install-delegates-with-custom-certs/#install-with-custom-certificates ).
```

**Pull request merged**